### ISSUE TITLE
Build all groups using runtime solution

### ISSUE THREAD

**Issue by choweiyuan** [wro4j/wro4j#967]
Found this issue/suggestion here
http://stackoverflow.com/questions/24583490/wro4j-build-all-groups-using-runtime-solution

Would be great if we could use this out of the box, since CSS may take a while to process on first request (for my case is around 5 secs)

**Comment by alexo** [comment]
There are several problems with this approach:
- building all groups using runtime solution could perform a high load during application start-up on your server (considering there are potentially large groups require costly processing). 
- It might be possible that some of the groups are not required ever (example: a page which loads a specific group which is not accessed ever). Using "lazy approach" is a better choice most of the times.
- It is possible to not know upfront all the groups (which might be created dynamically). In this situation, number of groups could be virtually infinite. As result - building all the groups doesn't scale.

The alternative solutions are the following:
- use build-time solution. It will build resources only once (no overhead of the first request).
- as part of the application start-up phase, request explicitly the resources which are taking long (a simple curl command with the resource url provided as an argument).

**Comment by alexo** [comment]
Let me know if the above comment is useful and if you agree with proposed solutions. 
For now, I'm closing this issue with WONTFIX resolution. However, you can reopen it if you don't agree.
